src/main/java/duke/ui/Ui.java

```diff
@@ -36,9 +36,13 @@ public static void printNoTaskFound() {
     }
 
     public static void printInvalidCommand() {
-        System.out.println("Command is invalid, please try again!");
+        System.out.println("☹ OOPS!!! command is invalid, please try again!");
     }
 
+    public static void printFindResultHeader(String keyword) {
+        System.out.println("Here are the matching tasks in your list containing "
+                + "\"" + keyword + "\"" + ":");
+    }
```

Comment:
I like the structure of your codes, but it can be too lengthy when writing strings. 
I see that most of your methods prints our similar strings. Probably can try to use getMessage() method to shorten and make your strings a little more consistent. Great work overall!!